fdbserver/DataDistribution.actor.h

```diff
@@ -114,6 +114,22 @@ struct GetTeamRequest {
 
 	GetTeamRequest() {}
 	GetTeamRequest( bool wantsNewServers, bool wantsTrueBest, bool preferLowerUtilization, double inflightPenalty = 1.0 ) : wantsNewServers( wantsNewServers ), wantsTrueBest( wantsTrueBest ), preferLowerUtilization( preferLowerUtilization ), inflightPenalty( inflightPenalty ) {}
+
+	std::string getDesc() {
```

Comment:
Is this function used anywhere?

Reply:
It was used in the commented out debug trace.